koku/api/report/aws/serializers.py

```diff
@@ -96,6 +108,13 @@ def validate_group_by(self, value):
 
         """
         validate_field(self, "group_by", GroupBySerializer, value, tag_keys=self.tag_keys)
+        # Additionally, since we only have the org_unit_id group_by available for cost reports
+        # we must explicitly raise a validation error if it is a different report type
+        if "org_unit_id" in self.initial_data.get("group_by", {}).keys():
+            request = self.context.get("request")
+            if "costs" not in request.path:
```

Comment:
@abaiken probably removing this is all that would be needed if we ever want to support `org_unit_id` filter/group_by in the instance-type or storage API?

Reply:
Possibly - but we would have to make sure that the total sum [function](https://github.com/project-koku/koku/pull/2139/files#diff-9fa2e6770cbc217b865ad5a5f33ce5bcR450) that adds together the query sums works for the structure of the query sums returned by the storage and instance-type endpoints. The filter works for those endpoints - it is only the group by that is disabled.